src/main/java/teammates/ui/controller/StudentProfilePageData.java

```diff
@@ -1,14 +1,45 @@
 package teammates.ui.controller;
 
 import teammates.common.datatransfer.AccountAttributes;
+import teammates.common.datatransfer.StudentProfileAttributes;
+import teammates.common.util.Const;
+import teammates.ui.template.StudentProfileEditBox;
+import teammates.ui.template.StudentProfileUploadPhotoModal;
 
 public class StudentProfilePageData extends PageData {
 
-    public String editPicture;
-
+    private StudentProfileEditBox profileEditBox;
+    private StudentProfileUploadPhotoModal uploadPhotoModal;
+    
     public StudentProfilePageData(AccountAttributes account, String editPicture) {
```

Comment:
This variable is used to indicate whether a new photo has just been uploaded.
I can change it to `isEditingPicture` or `isEditingPhoto` in methods, but for the JSTL template I can't name variables starting with 'is' (otherwise I need to set the getter as `getIs...` which is ugly).